src/main/java/frc/robot/subsystems/intake/commands/IntakeByVision.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+import java.util.function.BooleanSupplier;
+
+/**
+ * Intakes only Cargo colored as the robot's alliance color.
+ */
+public class IntakeByVision extends CommandBase {

```

Comment:
What is the difference between this command to `BasicIntakeCommand`?
It seems like they are doing the same.